src/app/_entities/slack-team/slack-team.ts

```diff
@@ -11,7 +11,7 @@ export class SlackTeam extends SlackTeamModel {
         entity.updatedBy = model.updatedBy;
         entity.teamId = model.teamId;
         entity.teamName = model.teamName;
-        entity.publicationChannel = model.publicationChannel;
```

Comment:
We can keep this attribute, it may be useful in the future to display the name of the channel.